xml/System.Reflection.PortableExecutable/DllCharacteristics.xml

```diff
@@ -49,7 +49,7 @@
       </ReturnValue>
```

Comment:
Can we add a type summary, something like "Describes the characteristics of a dynamic link library."?

Reply:
The type summary sounds good to me, I will apply it.
@GrabYourPitchforks @steveharter as the area owners, do you agree with the type summary?

Reply:
Actually, @tmat please take a look at this PR as well.

Reply:
lgtm